### PR TITLE
LPS-156293 Replace `href="javascript:;"` with `href="javascript:void(…

### PR DESCRIPTION
**Pull request by georgel-pop-lr**
…0);"` for better compatibility between browsers

### CODE REVIEW

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

File: modules/util/portal-tools-soy-builder/src/test/resources/com/liferay/portal/tools/soy/builder/commands/dependencies/build_soy/expected/text_localizable.soy.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ soy.$$registerDelegateFn(soy.$$getDelTemplateId('ddm.field'), 'text_localizable'
...
...playValue__soy5) + '</textarea>' : '<input aria-describedby="' + soy.$$escapeHtmlAttribute(opt_data.name) + '_desc" class="field form-control language-value" dir="' + soy.$$escapeHtmlAttribute(opt_data.dir) + '" id="' + soy.$$escapeHtmlAttribute(opt_data.name) + '" name="' + soy.$$escapeHtmlAttribute(opt_data.name) + '" placeholder="' + soy.$$escapeHtmlAttribute(opt_data.placeholder) + '" ' + ((opt_data.readOnly) ? 'readonly' : '') + ' type="text" value="' + soy.$$escapeHtmlAttribute(displayValue__soy5) + '">') + '<div class="input-localized-content hidden" id="' + soy.$$escapeHtmlAttribute(opt_data.name) + 'ContentBox" role="menu">' + ((opt_data.availableLocalesMetadata) ? ddm.available_locales(opt_data) : '') + '</div>' + ((opt_data.tooltip) ? '<span class="input-group-addon"><span class="input-group-addon-content"><a class="help-icon help-icon-default icon-monospaced icon-question" data-original-title="' + soy.$$escapeHtmlAttribute(opt_data.tooltip) + '" href="javascript:void(0);" title="' + soy.$$escapeHtmlAttribute(opt_data.tooltip) + '"></a></span></span>' : '') + '</div>' + ((opt_data.childElementsHTML) ? soy.$$filterNoAutoescape(opt_data.childElementsHTML) : '') + '</div>';
```
Comment (diegonvs) [inline modules/util/portal-tools-soy-builder/src/test/resources/com/liferay/portal/tools/soy/builder/commands/dependencies/build_soy/expected/text_localizable.soy.js:25]: Hey @georgel-pop-lr, you touched a generated file here. If you caught any CI errors, I think you may take a look at this

Comment (georgel-pop-lr) [reply modules/util/portal-tools-soy-builder/src/test/resources/com/liferay/portal/tools/soy/builder/commands/dependencies/build_soy/expected/text_localizable.soy.js:25]: Hi @diegonvs,

Thanks for looking, I see now he stable tests passed.

As there are many files affected, could we ask some help from someone on QA? 

Regards.

Comment (diegonvs) [reply modules/util/portal-tools-soy-builder/src/test/resources/com/liferay/portal/tools/soy/builder/commands/dependencies/build_soy/expected/text_localizable.soy.js:25]: > Thanks for looking, I see now the stable tests passed.



> As there are many files affected, could we ask some help from someone on QA?

Yes, @john-co, taking a look on LocalFile.FrontendDataSet#ActionsCanBeCustomized, It looks like we haven´t modified anything there. Same for LocalFile.FormsColorField#PredefinedValueCanBeAdded

Comment (diegonvs) [reply modules/util/portal-tools-soy-builder/src/test/resources/com/liferay/portal/tools/soy/builder/commands/dependencies/build_soy/expected/text_localizable.soy.js:25]: Same for [LocalFile.OpenIDConnect#OpenIDConnectSmokeWithAuth0](https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/32207//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_OpenIDConnect_OpenIDConnectSmokeWithAuth0_)

**Comment by john-co** [comment]
> As there are many files affected, could we ask some help from someone on QA?
> 
> Yes, @john-co, taking a look on LocalFile.FrontendDataSet#ActionsCanBeCustomized, It looks like we haven´t modified anything there. Same for LocalFile.FormsColorField#PredefinedValueCanBeAdded
>
> Same for [LocalFile.OpenIDConnect#OpenIDConnectSmokeWithAuth0](https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/32207//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_OpenIDConnect_OpenIDConnectSmokeWithAuth0_)

cc/ @mateusralv , can you analyze the test results here?

**Comment by mateusralv** [comment]
Taking a look 🔎

**Comment by mateusralv** [comment]
Hey @georgel-pop-lr and @diegonvs, the tests are failing don't seem to be related to this PR. 

cc: @john-co

**Comment by georgel-pop-lr** [comment]
> Hey @georgel-pop-lr and @diegonvs, the tests are failing don't seem to be related to this PR.
> 
> cc: @john-co

Hi @mateusralv,

Thanks for checking.

@diegonvs could we forward this? 

Regards.

**Comment by diegonvs** [comment]
Yes, we can

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Pull request closed without merging**